master/networking/ipv6.md

```diff
@@ -110,8 +110,13 @@ steps below.
 
 If you wish to only use IPv6 (by disabling IPv4) or your hosts only have
 IPv6 addresses, you must disable autodetection of IPv4 by setting `IP`
-to `none`.  With that set you must also pass a `CALICO_ROUTER_ID` to each
-calico-node pod.
+to `none`.
+
+With IPv4 disabled, you must configure another method to calculate the BGP router ID. There are
```

Comment:
Is it worth an explanatory sentence along the lines of "When IPv4 is enabled, calico sets the router ID to the IP of the host."

Reply:
We could - I almost did but thought it was more wordy than needed, and perhaps exposing more implementation detail than needed for this section, since this is just about "How do I make IPv6 work".

Reply:
Well you're kinda hinting at it with "you must configure **another** method"

Reply:
Updated